rosflight_firmware/CMakeLists.txt

```diff
@@ -16,23 +16,17 @@ if(NOT EXISTS "${FIRMWARE_SUBMODULE_DIR}/.git")
   )
 endif()
 
-# git version hash
-execute_process(
-  COMMAND git rev-parse --short=8 HEAD
-  WORKING_DIRECTORY ${FIRMWARE_SUBMODULE_DIR}
-  OUTPUT_VARIABLE GIT_VERSION_HASH
-  OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
-# git version string
-execute_process(
-  COMMAND git describe --tags --abbrev=8 --always --dirty --long
-  WORKING_DIRECTORY ${FIRMWARE_SUBMODULE_DIR}
-  OUTPUT_VARIABLE GIT_VERSION_STRING
-  OUTPUT_STRIP_TRAILING_WHITESPACE
-)
+# Git information
+execute_process(COMMAND git log -1 --format=%h
```

Comment:
just curious -- what's the difference?

Reply:
There is no difference.  I was just trying to debug the `GIT_VERSION_HASH=0x00` thing, and tried this method.  It turns out that it was something else, but I forgot to change this back.